modules/core/test/v2/unit/wallet.ts

```diff
@@ -857,6 +857,57 @@ describe('V2 Wallet:', function() {
 
       response.isDone().should.be.true();
     });
+
+    it('should create algo non participating key reg transaction', async function() {
```

Comment:
Also should be in a coin specific test file.

Reply:
It has been moved in coin specific test file